db/schema.rb

```diff
@@ -366,6 +366,7 @@
     t.boolean "eligible_for_funding", default: false, null: false
     t.text "funding_choice"
     t.text "nino"
+    t.text "status", default: "pending", null: false
```

Comment:
There are too many statuses to my liking :( Participant profiles have statuses, too, active or withdrawn.

Can we name it `lead_provider_approval_status`, or something like that?

Reply:
this has now been renamed to `lead_provider_approval_status` for lead provider facing side eg in the api it is still displayed as `status`